File: testdata/dummy.go
package testdata

var okayVar = "some stuff"

var internalSecret string = "asdfasdfasdf"

var (
	anotherOkayOne = "blah"
	authToken      = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"
)

const (
	AccessCode     = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"
	API_KEY_HEADER = "X-Api-Key"
)

var RealPostgresUri = "postgres://myuser:password123@blah.com:5432/mydb?sslmode=disable"
var TestPostgresUri = "postgres://myuser:password123@localhost:5432/mydb?sslmode=disable"

/*
Multiline comment
postgres://myuser:password123@somepostgresdb:5432/mydb?sslmode=disable
*/

func blah() {
	badPassword := "stupid"

	emptyPassword := ""

	if badPassword == emptyPassword {
		//do nothing
	}
}

type Config struct {
	Secret string
}

var conf = Config{
	Secret: "blah",
}

var PasswordFormat = "([0-9]+):(.+)"

var StatusAlternatives = "Alternatives" //variable name contains sAlt. Should not match the "salt" pattern

var blahToken = "password"

// blop is a function that does nothing
func blop() {
	// this is a local comment
	// "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"
}

File: testdata/dummy.java
package com.whatever.something;

import org.apache.commons.lang3.StringUtils;
import org.apache.ws.security.WSConstants;
import org.apache.ws.security.WSEncryptionPart;
import org.apache.ws.security.WSSConfig;
import org.apache.ws.security.WSSecurityException;

public class SomethingHandler implements SOAPHandler<SOAPMessageContext> {
    private String someTokenPassword = "AERWEk33se"; // default value

    public SomethingHandler(ServiceProperties properties) throws FileNotFoundException {
        this.serviceProperties = properties;
        // initialize configurable properties
        populateProperties();
        // initialize crypto
        createCryptoProperties();
    }

    const String INTERNAL_API_KEY = 'kiu#pKJSDK;LE';
    // this is a local comment
    // "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"

    public blah() {
        NewStaticCredentials("AKIAYTHMXXXGSVYYYWE6", "rP22kgSajDwOyWVU/iiii1UEdJk333QUbxwtiVCe");
    }
}

/*
 * This is a multiline comment
 * it contains postgres://myuser:password123@somepostgresdb:5432/mydb?sslmode=disable
 */
